ivy/functional/backends/paddle/sorting.py

```diff
@@ -122,3 +122,18 @@ def msort(
     a: Union[paddle.Tensor, list, tuple], /, *, out: Optional[paddle.Tensor] = None
 ) -> paddle.Tensor:
     return paddle.sort(a, axis=0)
+
+
+@with_unsupported_device_and_dtypes(
+    {"2.5.0 and below": {"cpu": ("complex64", "complex128")}},
+    backend_version,
+)
+def argpartition(
+    x: paddle.Tensor,
+    /,
+    kth: Union[int, paddle.Tensor],
+    *,
+    axis: int = -1,
+    order: Optional[str] = None,
+) -> paddle.Tensor:
+    return paddle.argsort(x, axis=axis).argsort(axis=axis)
```

Comment:
`argpartition` and `argsort` is not the same, [reference](https://www.pythonpool.com/numpy-argpartition/)